### ISSUE TITLE
it is impossible for me to compile 

### ISSUE THREAD

**Issue by demianrey** [fuzzy69/urlchecker#1]
Hello, first of all thanks for your work but I have not been able to compile and start using it, it is a tool that I really need ... is it possible that you can upload the already compiled binary? 
Regards

**Comment by fuzzy69** [comment]
Hi Demian,
thanks for the report, please let me know about your OS details, I presume it's a Windows host, Also, if you have a compiler output error log, please attach it here.
Unfortunately, compile process is not streamlined yet, it requires some separate steps like compiling HTML Tidy shared/dynamic library separately and moving built file into a build tree after wise, ideally it should all be done by CMake and without any manual intervention.
I'll try to build an executable binary when I find some free time, probably in a few days. Also, please note that the project is still in progress and there are some annoying bugs and bad designs, but I think it's usable for some things even in this state.

**Comment by fuzzy69** [comment]
Added a first release for Win64 platform, I'll update project files in a day or two.

**Comment by demianrey** [comment]
Hello, sorry for not answering before ... when I tried to compile it was on an Ubuntu system but it gave me an error from the QT5 libraries (I just didn't take a screenshot). Thanks for uploading it compiled for Windows, I'm going to review it and tell you how it went, thanks again.
Regards

**Comment by fuzzy69** [comment]
Yeah, I'll need an error log anyway, it could be anything. I suspected that it was some other library (probably HTML Tidy) which requires a separate step in process, and it fails on Windows of course.
I fixed it somehow to compile on Windows, but I want to automate the whole process before pushing the change, so it could be done by just CMake configure and build. You could try to compile it from Qt Creator (or KDevelop), maybe it'll pick up some missing library along the way.
Also, I'm not very impressed with how it works on Windows, it doesn't work great on *nix as well (yet), but it somehow "laggy" there. Hopefully it'll be improved soon.
Thanks for the feedback, hopefully you'll be able to run it and give more feedback so I can improve it further ;-).
